src/containers/StructureOutputContainer.js

```diff
@@ -68,24 +68,23 @@ const StructureOutputContainer = (props) => {
       className="structure-section"
       data-testid="structure-output-section"
     >
-      {manifestFetched && smData != null && (
-        <div data-testid="structure-output-list">
-          <List items={smData} />
-          <Row>
+      <div data-testid="structure-output-list">
+        <List items={smData} />
+        {!props.disableSave && 
+          (<Row>
```

Comment:
Are the `manifestFetched` and `smData` guards still needed?

Reply:
Yes! It may have overwritten when rebasing. Good catch :100: